pkg/constchecker/testdata/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all
+
+
+all: bpf_foo.o
+
+bpf_foo.ll: bpf_foo.c
+	clang -I../../../bpf/include -O2 -g -target bpf -emit-llvm -Wall -Werror -Wno-address-of-packed-member -Wno-unknown-warning-option -c bpf_foo.c -o bpf_foo.ll
+
+bpf_foo.o: bpf_foo.ll
+	llc -march=bpf -mcpu=probe -mattr=dwarfris -filetype=obj -o bpf_foo.o bpf_foo.ll
+
```

Comment:
When is these targets executed? Can't we make this part of the release tests? Do we need to include the `bpf_foo.o` into the tree?

Reply:
I think it's simpler if we do just keep the `bpf_foo.o` in the tree. It is not supposed to change, unless we add more tests (e.g., if we want to support additional types). The `Makefile` was just for my convenience during development.

Reply:
I think if it's not too much complexity I would prefer if we don't check-in a precompiled .o file, also as we might upgrade our internal clang/llc build for the Cilium image (https://github.com/cilium/image-tools/tree/master/images/llvm), so it would be good to then build bpf_foo.c with the same compiler binaries which also build Cilium's BPF programs.

Reply:
OK, I removed the .o file. Instead, I pass the object file to the constchecker test via an environment variable, which what I also do for the loadbalancer test.